pkg/manager/manager.go

```diff
@@ -23,21 +25,23 @@ type MediaManager struct {
 	tmdb    TMDBClientInterface
 	indexer IndexerStore
 	library library.Library
+	storage storage.Storage
 }
 
-func New(tmbdClient TMDBClientInterface, prowlarrClient ProwlarrClientInterface, library library.Library) MediaManager {
+func New(tmbdClient TMDBClientInterface, prowlarrClient ProwlarrClientInterface, library library.Library, storage storage.Storage) MediaManager {
 	return MediaManager{
 		tmdb:    tmbdClient,
 		indexer: NewIndexerStore(prowlarrClient),
 		library: library,
+		storage: storage,
 	}
 }
 
 type SearchMediaResponse struct {
 	Page         *int                 `json:"page,omitempty"`
-	Results      []*SearchMediaResult `json:"results,omitempty"`
 	TotalPages   *int                 `json:"total_pages,omitempty"`
 	TotalResults *int                 `json:"total_results,omitempty"`
+	Results      []*SearchMediaResult `json:"results,omitempty"`
```

Comment:
Did something automatically move these fields around?

Reply:
I don't remember what but there is a linter that checks ordering on the structs to save memory size and it complains if the ordering isn't right.. but its not an automatic thing I have to fix it manually